### PR TITLE
Revamps wasteplanet_unhonorable

### PR DESCRIPTION
**Pull request by Dethstorm**
## About The Pull Request
Revamps all of wasteplanet_unhonorable to not be a mess of a ruin.
Screenshot_2370
Screenshot_2371

## Why It's Good For The Game
Because bad ruins are bad.

## Changelog

:cl:
add: new stuff to wasteplanet_unhonorable
add: new areas for wasteplanet_unhonorable
/:cl:

### CODE REVIEW

**Comment by Dethstorm** [comment]
Made a PR for the sole reason of knowing what's causing conflicts.

File: code/game/area/areas/ruins/wasteplanet.dm [hunk 1]
```diff
@@ -29,3 +29,20 @@
 /area/ruin/wasteplanet/abandoned_mechbay/engineering
 	name = "Abandoned Mechbay Engineering"
 	icon_state = "engine"
+
+//Abandoned Waste Site
+
+/area/ruin/wasteplanet/wasteplanet_radiation/main
+	name = "Abandoned Waste Site"
+	requires_power = TRUE
```
Comment (Erikafox) [inline code/game/area/areas/ruins/wasteplanet.dm]:
```suggestion
```

File: code/game/area/areas/ruins/wasteplanet.dm [hunk 2]
```diff
@@ -29,3 +29,20 @@
...
+	icon_state = "green"
+
+/area/ruin/wasteplanet/wasteplanet_radiation/maint
+	requires_power = TRUE
```
Comment (Erikafox) [inline code/game/area/areas/ruins/wasteplanet.dm]:
```suggestion
```

File: code/game/area/areas/ruins/wasteplanet.dm [hunk 3]
```diff
@@ -29,3 +29,20 @@
...
+	name = "Abandoned Maintenance Area"
+	icon_state = "engine"
+
+/area/ruin/wasteplanet/wasteplanet_radiation/containment
+	requires_power = TRUE
```
Comment (Erikafox) [inline code/game/area/areas/ruins/wasteplanet.dm]:
```suggestion
```

**Review by Erikafox — CHANGES_REQUESTED** [review]
Please remember to update the map catalog (_Maps/map_catalog.txt) to reflect the changes you've made here

The exterior of the area could use more fluff. I recommend putting down dirty decals, maybe some warning signs, a few light mobs (abandoned bots and maybe a hivebot or two), OR. Trim down what you have to just the ruin, and use turf passthrough + area passthrough to let the game set the ruin's surroundings up for you.

The area surrounding the ruin should be /overmap_encounter/planetoid/wasteplanet/explored, It'll let the area have weather.

image
replace these tiles with the wasteplanet variants. That should fix your main airlock not working
image
no need for cycle link helpers here or at the other airlock, the AAC handles it automatically.

Speaking of, the internal one is bugged, but I think that's alright here. It keeps players from going into radiation hell quickly.
StrongDMM_yLmwdjas28
Some more decalling would be bitchin, plus maybe some firelocks sprinkled under the doors.

It is certainly an improvement on the previous iteration, so congrats.

**Comment by Dethstorm** [comment]
Cycle link helpers were removed and tiles were changed, but I can't get the cycling airlock to work. Screenshots Soon TM?

**Review by Erikafox — DISMISSED** [review]
unless you intend for the outdoors to be unlit, use the /lit tile variants there
StrongDMM_XEEYwmDiAf
replace the tiles in here with good atmos tiles (the ones you use inside), may solve the issue with the AAC

How'd you swap the areas? I tested it on my local like such

and I didn't see any issues with it

**Comment by Dethstorm** [comment]
> unless you intend for the outdoors to be unlit, use the /lit tile variants there StrongDMM_XEEYwmDiAf replace the tiles in here with good atmos tiles (the ones you use inside), may solve the issue with the AAC
> 
That bit of the ruin doesn't really need lighting; it's abandoned and the light bulb's broken, don't see a reason to use /lit variants. Switched the tiles around, I think the AAC works now but there's one issue left; the pressure doesn't really rise:
Screenshot_2375

> How'd you swap the areas? I tested it on my local like such  and I didn't see any issues with it
> 
Areas were switched to "/area/ruin/wasteplanet/wasteplanet_radiation" from "area/overmap_encounter/planetoid/wasteplanet/explored". The latter caused any turfs marked with that area to be replaced with planet generation.
The only possible issue this might cause is the lack of weather in the area, but I don't think that's too big of a problem.

**Comment by Erikafox** [comment]
> > unless you intend for the outdoors to be unlit, use the /lit tile variants there StrongDMM_XEEYwmDiAf replace the tiles in here with good atmos tiles (the ones you use inside), may solve the issue with the AAC
> 
> That bit of the ruin doesn't really need lighting; it's abandoned and the light bulb's broken, don't see a reason to use /lit variants. Switched the tiles around, I think the AAC works now but there's one issue left; the pressure doesn't really rise: Screenshot_2375


That area is fine, I said the outdoors (so everything surrounding the ruin)

**Comment by Dethstorm** [comment]
Ah I think the outdoors should be fine as unlit

**Review by Erikafox — APPROVED** [review]

**Review by spockye — APPROVED** [review]
a 100% improvement, good work!

**Comment by spockye** [comment]
I'll Merge this once checks pass. 
Send me a message in ruinyard if you need any help!

**Comment by Dethstorm** [comment]
> I'll Merge this once checks pass. Send me a message in ruinyard if you need any help!

Oh I had forgotten about this. I'll check it once I get home. May or may not need help

(Edit: I forgot again)

**Comment by Dethstorm** [comment]
Ready for merging.

**Pull request merged**